lib/src/front_end/chain_builder.dart

```diff
@@ -230,11 +230,6 @@ class ChainBuilder {
         _visitTarget(expression);
 
       // Selectors.
-      case AssignmentExpression():
-        var piece = _visitor.createAssignment(expression.leftHandSide,
-            expression.operator, expression.rightHandSide);
-        _calls.add(ChainCall(piece, CallType.property));
-
```

Comment:
I think this was used back when cascades went through this code path, but they don't any longer, so it's just dead code.